src/components/structures/login/Registration.js

```diff
@@ -164,10 +164,22 @@ module.exports = React.createClass({
         if (!success) {
             let msg = response.message || response.toString();
             // can we give a better error message?
-            if (response.errcode == 'M_MAU_LIMIT_EXCEEDED') {
+            if (response.errcode == 'M_RESOURCE_LIMIT_EXCEEDED') {
```

Comment:
There seems to be a bit of duplication across `RoomStatusBar`, `Login`, `Registration` & `ServerLimitBar` for this use case, with the only thing different being the specific translation string to use. I wonder if it would make sense to put some of this somewhere centralized (e.g. a helper)?

Reply:
Yeah, it is a bit - hopefully this is a bit better now.